--- key-bindings.zsh.bk	2026-04-19 17:49:21.673291466 +0300
+++ key-bindings.zsh	2026-04-19 17:49:29.111367918 +0300
@@ -127,6 +127,7 @@
 
 # CTRL-R - Paste the selected command from history into the command line
 fzf-history-widget() {
+  fc -IR
   local selected extracted_with_perl=0
   setopt localoptions noglobsubst noposixbuiltins pipefail no_aliases no_glob no_sh_glob no_ksharrays extendedglob 2> /dev/null
   # Ensure the module is loaded if not already, and the required features, such
