diff --git a/synced/.config/git/scripts/git-prompt.sh b/synced/.config/git/scripts/git-prompt.sh
index 4dcf4df..a8efc93 100644
--- a/synced/.config/git/scripts/git-prompt.sh
+++ b/synced/.config/git/scripts/git-prompt.sh
@@ -627,7 +627,7 @@ __git_ps1 ()
 
 		if [ -n "${GIT_PS1_SHOWUNTRACKEDFILES-}" ] &&
 		   [ "$(git config --bool bash.showUntrackedFiles)" != "false" ] &&
-		   git ls-files --others --exclude-standard --directory --no-empty-directory --error-unmatch -- ':/*' >/dev/null 2>/dev/null
+		   [ "$(git status --porcelain 2>/dev/null | grep '^??' | head -n 1)" ]
 		then
 			u="%${ZSH_VERSION+%}"
 		fi
